ntbs-service/wwwroot/source/Components/ValidatePostcode.ts

```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue';
+import { getHeaders } from '../helpers';
+import axios from 'axios';
+
+const ValidatePostcode = Vue.extend({
+    props: ['shouldvalidatefull'],
+    methods: {
+      validate: function (event: FocusEvent) {
+          // Our onBlur validate events happen on input fields

```

Comment:
Definitely not a major issue, but there seems to be inconsistency in indentation here.

In VS Code `Ctrl + Shift + I` (as in **I**ndigo - font doesn't make that clear. EDIT: Font does make it clear in code formatting) formats document - think there's a way to make it format on save too.